javers-core/src/main/java/org/javers/common/properties/PropertyConfiguration.java

```diff
@@ -1,10 +1,11 @@
 package org.javers.common.properties;
 
-import java.util.Properties;
 import org.javers.common.exception.JaversException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Properties;
+
```

Comment:
staramy się nie robimy takich zmian, które nic nie zmieniają, bo utrudnia to code review